.github/workflows/go.yml

```diff
@@ -8,6 +8,7 @@ on:
 jobs:
 
   build:
+    if: github.event.pull_request.draft == false
```

Comment:
Shouldn't this be `!=` ?

Reply:
No, why? You only want to run the action when the pull request is not draft, right? Currently, for draft it will look like`true == false`, i.e: `false`